src/main/resources/database/changes/release-16/pgcrypto_fix.sql

```diff
@@ -0,0 +1 @@
+DROP EXTENSION IF EXISTS pgcrypto CASCADE;
```

Comment:
Shouldn't need this anymore?

Reply:
I think we do, the next script tries to set it up.  Uses it altering the actionexporter.address table.